binaries/dump_operator_names.cc

```diff
@@ -0,0 +1,69 @@
+#include <torch/csrc/jit/script/module.h>
```

Comment:
Note that majority of files under binaries/ have "Copyright (c) 2016-present, Facebook, Inc." license header. Consider following this convention?